src/App.tsx

```diff
@@ -1,9 +1,8 @@
-import React from "react";
 import logo from "./logo.svg";
 import "./App.scss";
-import Counter from "./components/Counter";
+import Counter from "components/Counter";
 
-const App: React.FC = () => {
+const App = (): JSX.Element => {
```

Comment:
The react team suggests not using `React.FC` anymore: https://github.com/facebook/create-react-app/pull/8177